portal-survey-2023.Rmd

```diff
@@ -0,0 +1,291 @@
+   group_by(issue) %>%
+  count(response)
+
+grouped_bar(upload)
+```
+
+As with data upload, the majority and largest proportion of users thought "confusing or insufficient instructions" was a real issue (Figure \@ref(fig:fig-sharing-annotation)).
+
+#### Major focus area of improvement identified: 
+
+**Instructions could be improved for easier data annotation and upload** 
+
+```{r fig-sharing-annotation, fig.cap="Caption", layout="l-body-outset", fig.width=9, fig.height=7}
+annotation <- tidy_survey_ordinal %>%
+  filter(area == "ability to annotate") %>%
+  mutate(response = factor(response, levels = c("Made it impossible to annotate", "Made it hard to annotate", "This was not an issue", "Not applicable"))) %>%
+   group_by(issue) %>%
+  count(response)
+
+grouped_bar(annotation)
+```
+
+
+### User sentiment summary
+
+Our results indicated that there are some users being left behind in all key areas.
+
+Users were also asked to provide a rating to summarize their overall experience in each area.
+When comparing the response distribution for these different areas all together, we see somewhat interestingly that the responses tend to be bi-modal (Figure \@ref(fig:fig-all-distributions)).
+Users who respond appear to be really pretty dissatisfied or pretty satisfied.
+While there is no area that is truly terrible, data sharing does have the lowest overall rating, with metadata sharing through the Data Curator App does having the lowest experience rating of 3 (Table \@ref(tab:table-ratings)). 
+
+
+```{r fig-all-distributions, fig.label="", fig.width=6.5, fig.height=10, echo=F}
+
+h <- tidy_survey_quant %>%
+  mutate(area = gsub("\\(.*", "", area))
+
+ggplot(h, aes(x = rating)) +
+  geom_histogram(binwidth=1) +
+  facet_wrap(area ~ ., ncol = 1) +
+  theme_light()
+```
+
+
+```{r table-ratings, fig.label="", echo=F}
+
+medians <- tidy_survey_quant %>%
+  group_by(area) %>%
+  summarize(median = round(median(rating), 2))
+
+ gt(medians)
+
+```
+
+## Discussion on improvements 
+
+The survey results will be integrated into plans for continuous improvement in data discovery, sharing, and reuse as the NF data platform continues to expand and receive investment. 
+
+We identified that data sharing should be most prioritized. 
+The next prioritization chooses between data findability versus data reuse, and the survey results suggest that data findability affected more users. 
+Both search functionality and the user interface are issues to target for data findability. 
+Respondent comments also indicated that improving data accessibility to patients is also needed for the NF Data Portal.
+
+To improve the data sharing experience, the analysis has already suggested having clearer instructions and making sure they are seen by users. 
+However, user comments also gave the insight that instructions become an issue because the current order/decoupling of events for data sharing is unintuitive -- data annotation should be done with data upload, not after. 
```

Comment:
Can you add an anonymous quote to support this?

Reply:
Yep, added